packages/paste-website/src/pages/primitives/listbox-primitive/index.mdx

```diff
@@ -0,0 +1,322 @@
+| `first`        | `() => void`                                            | Moves focus to the first item                                                                                                                          |             |
+| `last`         | `() => void`                                            | Moves focus to the last item                                                                                                                           |             |
+| `items`        | `Item[]`                                                | Lists all the listbox items with their id, DOM ref, disabled state and groupId if any. It is updated when registerItem and unregisterItem are called.  |             |
+| `setCurrentId` | `Dispatch<SetStateAction<string \| null \| undefined>>` | Sets currentId. It only updates the currentId state without moving focus.                                                                              |             |
+| `focusable?`   | `boolean`                                               | When an element is disabled, it may still be focusable. It works similarly to readOnly on form elements. In this case, only aria-disabled will be set. |             |
+| `disabled?`    | `boolean`                                               | Same as the HTML attribute                                                                                                                             |             |
+| `baseId?`      | `string`                                                | ID that will serve as a base for all the items IDs                                                                                                     |             |
+| `currentId?`   | `string`                                                | The current focused item id                                                                                                                            |             |
+| `groups?`      | `Group[]`                                               | Lists all the composite groups with their id and DOM ref. It is updated when registerGroup and unregisterGroup are called.                             |             |
+| `orientation?` | `"horizontal" \| "vertical"`                            | Defines the orientation of the listbox                                                                                                                 | "vertical"` |
+| `variant?`     | `"single" \| "multiple"`                                | Defines the selection mode of the listbox                                                                                                              | "single"`   |
+
+##### ListboxPrimitiveGroup
+
+No props to show
+
+##### ListboxPrimitiveItem
+
+**Note:** Most required props are provided by spreading the returned state from `useListboxPrimitiveState`.
```

Comment:
Question (non-blocking): Should this also be a neutral callout?

Reply:
Not sure! Copied what was done in other primitives.